core/src/main/java/org/apache/accumulo/core/clientImpl/AccumuloClientImpl.java

```diff
@@ -376,9 +376,9 @@ public SaslOptions useSasl() {
     public ConnectionOptions batchWriterConfig(BatchWriterConfig batchWriterConfig) {
       setProperty(ClientProperty.BATCH_WRITER_MAX_MEMORY_BYTES, batchWriterConfig.getMaxMemory());
       setProperty(ClientProperty.BATCH_WRITER_MAX_LATENCY_SEC,
-          batchWriterConfig.getMaxLatency(TimeUnit.SECONDS));
+          batchWriterConfig.getMaxLatency(TimeUnit.MILLISECONDS) + "ms");
```

Comment:
For other props that now have a timeduration type did you check how those are used to see if units need to be added anywhere else?

Reply:
I'm working on it, thanks.  The ZK timeout is used in a lot more places.  You think I should create a method in ClientProperty that sets the units?  That might be better than having a bunch of concats.

Reply:
I created setTime() and setBytes() in 80675e6e85.  They are kinda nice.